servicecontrol/creating-config-file.md

```diff
@@ -320,6 +319,118 @@ This entry should be set to `false` if there is no external process reading mess
 
 See [Installation](installation.md) for details on how to set this at install time.
 
+## Usage Reporting
+
+### LicensingComponent/ServiceControlThroughputDataQueue
+
+The queue on which throughput data is received by the ServiceControl Error instance. This setting must match the equivalent [`Monitoring/ServiceControlThroughputDataQueue`](/servicecontrol/monitoring-instances/installation/creating-config-file.md#usage-reporting-monitoringservicecontrolthroughputdataqueue) setting for the Monitoring instance.
+
+In most instances these settings do not need to be modified.
+
+If running multiple setups of the Platform Tools (i.e. multiple versions of ServiceControl Error and Monitoring instances) then modify these settings so that the queue on each Monitoring instance is matched to the queue of its Error instance.
```

Comment:
```suggestion
If running multiple setups of the Platform Tools (i.e. multiple versions of ServiceControl error and monitoring instances) then modify these settings so that the queue on each monitoring instance is matched to the queue of its error instance.
```